modules/infoicon/Infoicon.story.mdx

```diff
@@ -10,3 +10,63 @@ import { InfoiconStory } from "./Infoicon.story.tsx";
     <InfoiconStory />
   </Story>
 </Preview>
+
+## Available Props
+
+```typescript
+interface IInfoIconProps {
```

Comment:
This is not the pattern we have with the html tables. But I actually like this, it might be more intuitive/ easy to read than what we had before! You should suggest this as the new pattern 🙂